arbeitszeit_web/api_controllers/get_plan_api_controller.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+from uuid import UUID
+
+from arbeitszeit_web.api_presenters.response_errors import BadRequest
+
+
+@dataclass
+class GetPlanApiController:
+    def format_input(self, plan_id: str) -> UUID:
```

Comment:
I think that this method is not named properly. It does not format any input but instead instead parse input fields into structured data (ideally a use case request object).